service/history/workflow/mutable_state_impl_restart_activity_test.go

```diff
@@ -0,0 +1,370 @@
+package workflow
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
```

Comment:
Nit: Would you mind separating the temporal (internal) imports from the external ones? Our normal convention for organizing the import block (as I've been informed) is:
```
import (
    // standard library imports

    // external, non-temporal imports

    // temporal imports
)
```

In some places we separate temporal imports from _internal_ temporal imports (same repo) but I was told that's the old style

Reply:
![image](https://github.com/temporalio/temporal/assets/2232524/923f1253-80bf-4da6-a025-81d3d4dd8006)

Reply:
Thanks.